package.json

```diff
@@ -10,6 +10,7 @@
   },
   "devDependencies": {
     "@antora/atlas-extension": "1.0.0-alpha.2",
+    "@asciidoctor/tabs": "^1.0.0-beta.6",
```

Comment:
🤔 For consistency with other dependencies declaration, I suggest to use a specific version or only allow patch auto-bump (prefix `~` instead of `^`)

Reply:
I'll set a strict version to be sure it is the same as bonita-documentation-site